SEMVER.md

```diff
@@ -0,0 +1,39 @@
+# Semantic versioning
+
+LUX gebruikt Semantic versioning (SemVer)[^1] voor de versies van releases.
+Dit is een wijd gebruikt versienummeringssysteem, dat in de basis uit drie versienummers bestaat, namelijk:
+
+> MAJOR.MINOR.PATCH
+
+Deze kunnen uitgebreid worden met optionele pre-release- en build-metadata labels.
+
+De verschillende elementen worden, volgens het officiele schema[^2], verhoogd als volgt:
+
+> 1. MAJOR wordt verhoogd bij incompatibele API-wijzigingen,
+> 1. MINOR wordt verhoogd bij het toevoegen van functionaliteit die compatibel is met de vorige versie, en
+> 1. PATCH wordt verhoogd bij compatibele bugfixes.
+
+Binnen LUX hebben we dit uitgebreidt en worden de versienummer als volgt verhoogd:
+
+<dl>
+<dt>MAJOR</dt>
+<dd>wordt verhoogd bij incompatibele én impactvolle (visuele) wijzigingen die bij gebruikers van LUX toegankelijkheids- of regressieproblemen kunnen veroorzaken,</dd>
+<dt>MINOR</dt>
```

Comment:
Ik zou de minor en patch hier alleen noemen als we afwijken van de standaard

Reply:
Alleen MAJOR staat er nu beschreven.